backend/resume_evaluations/schemas.py

```
@@ -0,0 +1,39 @@
+    """Individual field-level feedback for each scoring category."""
+
+    skills_match_feedback: str
+    experience_relevance_feedback: str
+    keyword_match_feedback: str
+    projects_feedback: str
+    education_feedback: str
+    formatting_feedback: str
+    additional_value_feedback: str
+
+
+class ResumeEvaluationRequest(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)